src/main/resources/static/assets/css/style.css

```diff
@@ -21673,7 +21673,6 @@ a.material-icons-shopping_cart:hover {
   display: block;
   position: absolute;
   text-align: left;
-  margin-top: 2.3125rem;
```

Comment:
@airborn agree. After rethink better way of modifying theme is through custom CSS.